ocpmodels/trainers/md_trainer.py

```diff
@@ -134,27 +138,43 @@ def load_task(self):
             self.logger.log_plots(plots)
 
     # Takes in a new data source and generates predictions on it.
-    def predict(self, dataset_config, batch_size=32):
-        print(
-            "### Generating predictions on {}.".format(dataset_config["src"])
-        )
+    def predict(self, dataset_config, batch_size=32, train=True, verbose=True):
```

Comment:
Let's switch `train=True` to `mode="train"` here as well if you agree with my comment above.

And maybe also add a comment to describe what it does since it's a bit unintuitive why a `predict` function would have a `mode="train"`.

Reply:
Now that I think of it we probably don't need this argument entirely. This flag is to avoid the dataloader being constructed for train, val, test. Since none of that is necessary in a predict call (everything is in one dataloader) we can probably avoid this. Unless you can think of a situation where we may need it, but like you said it's pretty unintuitive?